FileTypes/src/main/java/com/infomaniak/library/filetypes/FileType.kt

```diff
@@ -46,9 +46,14 @@ enum class FileType(val icon: ImageVector, private val colorLight: Color, privat
     }
 
     companion object {
+        fun getMimeTypeFromFileName(fileName: String): String? {
+            return fileName.extractExtension()?.let { extension ->
+                MimeTypeMap.getSingleton().getMimeTypeFromExtension(extension)
+            }
+        }
+
```

Comment:
Maybe rename it to `guess` instead of `get` and add a TODO to move it into Core2. When we have a correct structure in Core2 we should move this code in the common part of Core2 maybe as it's kind of independent of FileType